### PR TITLE
[PWA-836] Localize Cart

### PR DESCRIPTION
**Pull request by anthoula**
## Description

Translate strings within Cart

## Related Issue

https://jira.corp.magento.com/browse/PWA-836

### Verification Stakeholders

@dpatil-magento 
@davemacaulay 
@jimbo 
### Verification Steps

1. Add a product to cart
2. Go to Cart page
3. Verify strings are translated where applicable
4. Remove all items from cart
5. Go to Cart page
6. Verify strings are translated where applicable

## Checklist

- [x] I have added tests to cover my changes, if necessary.
* I have updated the documentation accordingly, if necessary.

### CODE REVIEW

**Review by davemacaulay — DISMISSED** [review]

File: packages/venia-ui/lib/components/Country/country.js [hunk 1]
```diff
@@ -15,7 +16,14 @@ const Country = props => {
         }
     });
     const { countries, loading } = talonProps;
-    const { classes: propClasses, field, label, ...inputProps } = props;
+    const {
+        classes: propClasses,
+        field,
+        label,
+        translationId,
```
Comment (jimbo) [inline packages/venia-ui/lib/components/Country/country.js:23]: Good choice here to take this in as a prop. Naming looks good, too, though I wonder what happens if we need more than one.

File: packages/venia-ui/lib/components/CartPage/PriceSummary/priceSummary.js [hunk 2]
```diff
@@ -127,7 +142,15 @@ const PriceSummary = props => {
...
-                    {isCheckout ? 'Total' : 'Estimated Total'}
+                    {isCheckout
+                        ? formatMessage({
+                              id: 'priceSummary.total',
+                              defaultMessage: 'Total'
+                          })
+                        : formatMessage({
+                              id: 'priceSummary.estimatedTotal',
+                              defaultMessage: 'Estimated Total'
+                          })}
```
[the remark this answers is not in the document]
Comment (anthoula) [reply packages/venia-ui/lib/components/CartPage/PriceSummary/priceSummary.js]: Found and updated a few instances of this

[the remark this answers is not in the document]
Comment (jimbo) [reply packages/venia-ui/lib/components/CartPage/PriceSummary/priceSummary.js]: Thank you. 👍

**Review by jimbo — APPROVED** [review]

**Comment by dpatil-magento** [comment]
@anthoula PR looks good except `out-of-stock` string.

image

**Review by davemacaulay — APPROVED** [review]

**Review by jimbo — APPROVED** [review]

**Pull request merged**